packages/telegram-stamper/src/__tests__/stamp-test.ts

```diff
@@ -0,0 +1,41 @@
+import { test, expect } from "@jest/globals";
+import { TelegramStamper } from "../index";
+import { readFixture } from "../__fixtures__/shared";
+import { assertValidSignature } from "./shared";
+
+test("uses provided signature to make stamp", async function () {
+  const { privateKey, publicKey, pemPublicKey } = await readFixture();
+
+  const stamper = new TelegramStamper({
+    apiPublicKey: publicKey,
+    apiPrivateKey: privateKey,
+  });
+  // ToDo: init fails since there isn't an actual Telegram environment here
+  // await stamper.init()
```

Comment:
You can provide a fake TG storage by aliasing `window.Telegram.WebApp.CloudStorage.setItem` to `window.localStorage.setItem` (and likewise for get). That will let your tests pass and actually test the init code you wrote up until the point where the TG API is invoked.

Reply:
Added a mock local storage here to test this functinoality!